Common/src/grid_movement/CVolumetricMovement.cpp

```diff
@@ -293,6 +296,141 @@ void CVolumetricMovement::ComputeDeforming_Element_Volume(CGeometry *geometry, s
 
 }
 
+void CVolumetricMovement::ComputenNonconvexElements(CGeometry *geometry, bool Screen_Output) {
+  unsigned long iElem, PointCorners[8];
```

Comment:
If this routine is not super expensive we should include it in the mesh quality metrics.